### ISSUE TITLE
Current respository name and remote location details are wrong

### ISSUE THREAD

**Issue by patcorwin** [desktop/desktop#11155]
### Describe the bug

For two repos, when I choose either, it displays the wrong name in the repository menu, and displays the wrong location on github when I go to "Respository Settings".

It does appear, internally, to know the correct remote locations because "Changes" and "History" display the right things.

### Version & OS

Win10 2.6.0

### Steps to reproduce the behavior

I have no idea how or when this happened.  My only guess is I did a commandline `git add -A .` in one of the repos to detect renames.

### Expected behavior

The repository name is displayed correctly in "Current repository", the repository list and the remote location is displayed correctly in "Respository Settings".

### Actual behavior

As you can see in this gif, then name and remote location are all the same but the location on disk is correct, only one should be "mayaHooks".

### Screenshots

![RepoNames](https://user-images.githubusercontent.com/10981431/100728670-b0cb3200-337c-11eb-958a-79d3e53b3625.gif)

### Logs

Attach your logs by opening the `Help` menu and selecting `Show Logs...`, if applicable.

### Additional context

I plan on removing and adding it back but wanted to report it first in case there was some useful info.

**Comment by steveward** [comment]
@patcorwin thanks for the report. Just to clarify -- are these three separate remote repositories on GitHub? How were they added to GitHub Desktop? This sounds similar to some of the behavior we've seen reported in https://github.com/desktop/desktop/issues/3855. Usually clearing out `%AppData%\GitHub Desktop` and restarting GitHub Desktop resolves this issue.

**Comment by patcorwin** [comment]
Thanks, clearing out `%AppData%\Github Desktop` worked and they show up with the correct names.  FWIW, there are 3 local repos that point to two remote repos, I was super lazy and didn't clear out local files when rearranging things.  I can't remember how they were added, it was over three years ago, but I'm guessing I initialized the repo on github, then added local files.  I'm not sure exactly when this problem cropped up but I'd guess within the last few months.  Again, it appears fixed and I just wanted to address your follow up questions and say thanks.
